lib/new_relic/agent/instrumentation/roda/ignorer.rb

```diff
@@ -0,0 +1,46 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module Roda
+    module Ignorer
+      def self.should_ignore?(app, type)
+        return false if !app.opts.include?(:newrelic_ignores)
+
+        app.opts[:newrelic_ignores][type].any? do |pattern|
+          pattern === app.request.path_info
```

Comment:
Out of curiosity, why did you choose to use `===`?

Reply:
Sometimes `pattern` is a string when a user specifies a route. But if they don't specify a route, then `pattern` becomes a regexp `/.*/`. 

So sometimes we get `string === string`. And in string land `==` and `===` are alias of one another.

But in regexp land, `==` is for comparing regexps and `===` is for comparing regexps to strings, which `app.request.path_info` should always be. 

I think there may have been a more clever way to write this by always turning strings into regexps, but this is where I landed. Open to alt ideas ◡̈

Reply:
I think this is a very clever solution! Thanks for breaking it down! 🙂